packages/atomic/src/components/common/search-box/search-text-area.tsx

```diff
@@ -0,0 +1,140 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {JSXBase} from '@stencil/core/internal';
+import {AnyBindings} from '../interface/bindings';
+import {TextAreaClearButton} from './text-area-clear-button';
+
+interface Props extends JSXBase.TextareaHTMLAttributes<HTMLTextAreaElement> {
+  textAreaRef: HTMLTextAreaElement;
+  loading: boolean;
+  bindings: AnyBindings;
+  value: string;
+  ariaLabel: string;
+  onClear(): void;
+  popup?: {
+    id: string;
+    activeDescendant: string;
+    expanded: boolean;
+    hasSuggestions: boolean;
+  };
+}
+
+function getPopupAttributes(props: Required<Props>['popup']) {
+  return {
+    role: 'combobox',
+    autocomplete: 'off',
+    autocapitalize: 'off',
+    autocorrect: 'off',
+    'aria-activedescendant': props.activeDescendant,
+    'aria-expanded': `${props.hasSuggestions && props.expanded}`,
+    'aria-autocomplete': 'both',
+    'aria-haspopup': 'true',
+    'aria-controls': props.id,
+  };
+}
+
+function syncTextWithReplica(elem: HTMLTextAreaElement) {
+  const parent = elem.parentNode as HTMLElement;
+  if (elem.value === '\n') {
+    return;
+  }
+  if (parent) {
+    parent.dataset.replicatedValue = elem.value;
+  }
+}
+
+function resetReplicaText(elem: HTMLTextAreaElement) {
+  const parent = elem.parentNode as HTMLElement;
+  if (parent) {
+    delete parent.dataset.replicatedValue;
+  }
+}
+
+function collapseTextArea(elem: HTMLTextAreaElement) {
+  const parent = elem.parentNode as HTMLElement;
+  if (parent) {
+    parent.classList.remove('expanded');
+  }
+}
+
+function expandTextArea(elem: HTMLTextAreaElement) {
+  const parent = elem.parentNode as HTMLElement;
+  if (parent) {
+    parent.classList.add('expanded');
+  }
+}
+
+export const SearchTextArea: FunctionalComponent<Props> = ({
+  textAreaRef,
+  loading,
+  bindings,
+  onInput,
+  onFocus,
+  onBlur,
+  onChange,
+  onKeyDown,
+  value,
+  ariaLabel,
+  onClear,
+  popup,
+  ...defaultInputProps
+}) => (
+  <div class="grow flex overflow-hidden">
+    <div class="grow-wrap grow grid overflow-hidden">
+      <textarea
+        part="textarea"
+        aria-label={ariaLabel}
+        placeholder={bindings.i18n.t('search')}
+        class="placeholder-neutral-dark"
+        rows={1}
+        onInput={(e) => {
+          onInput?.(e);
+          syncTextWithReplica(e.target as HTMLTextAreaElement);
+        }}
+        onKeyDown={(e) => {
+          onKeyDown?.(e);
+          if (['Enter'].includes(e.key)) {
```

Comment:
Unless I'm missing something, I don't think should treat a string comparison like an array.
```suggestion
          if (e.key === 'Enter') {
```

Reply:
yeah that's fair i had 2 strings in there before so the array made more sense.